### DISCUSSION TITLE
Route [filament.company.resources.media.edit] not defined.

### DISCUSSION THREAD

**Discussion by kuperdigital** [awcodes/filament-curator#357] [Q&A]
Hi

I am running into this issue after I have uploaded an image with curator and clicked "Insert".
It gives the error:  Route [filament.company.resources.media.edit] not defined.

I am using curator in a form inside a filament panel.
I have a panel called 'company', inside that panel I have a page called 'Invoice' and then the form with the CuratorPicker. 

I guess I am doing something wrong. Any suggestions on what to look for?

**Comment by awcodes** [comment]
You need to add `CuratorPlugin::make()` to the ->plugins() modifier on your panel.

**Reply by kuperdigital** [reply]
I have this in my panel already:

```php
return $panel
            ->default()
            ->id('company')
            ->path('company')
            ->login(Login::class)
            ->registration(Register::class)
            ->passwordReset()
            ->homeUrl(static fn (): string => url(Pages\Dashboard::getUrl(panel: 'company', tenant: Auth::user()?->personalCompany())))
            ->plugin(
                \Awcodes\Curator\CuratorPlugin::make()
                    ->label('Media')
                    ->pluralLabel('Media')
                    ->navigationIcon('heroicon-o-photo')
                    ->navigationGroup('Content')
                    ->navigationSort(3)
                    ->navigationCountBadge()
            )
```

**Reply by awcodes** [reply]
That's the only reason I can think that it wouldn't be registered, unless your routes are cached.

**Reply by kuperdigital** [reply]
Hmm okay. The cache is cleared :-/

**Reply by awcodes** [reply]
Might have something to do with multi-tenancy too. But no one else has reported an issue with it.
